tests/skeletons.rs

```diff
@@ -530,7 +527,7 @@ name = "test-dummy"
 path = "src/main.rs"
 
 [dependencies]
-uuid = { version = "=0.8.0", features = ["v4"] }        
+either = { version = "=1.8.1" }        
```

Comment:
Same here—why the swap?

Reply:
`uuid` had multiple dependencies, which weren't reflected in the lockfile of the original test, and it thus couldn't be parsed with `cargo metadata`. I tried to find a package without any dependencies, to keep the lockfile content simple, otherwise it would be quite large, which in turn made the test workspace setup unnecessarily large.